ios/Runner/Info.plist

```diff
@@ -22,16 +22,21 @@
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSContactsUsageDescription</key>
+	<string>To send email with these contacts</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>To save photo to library, we need permission to access your photo library.</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>To select and upload photo, we need permission to access your photo library.</string>
+	<key>UIBackgroundModes</key>
```

Comment:
Did you forget to enable `Push notification`?

Reply:
@@ it's enabled but it doesn't need to be declared here